changelog.md

```diff
@@ -5,15 +5,14 @@
 We expect this release to ship in the DevTools of [Chrome 93](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within a day!
 
 ## Notable changes
-* The **Performance Category** had a number of scoring changes to align with other performance tools and to better reflect the state of the web:
-  - TODO: decide on FAQ incorporation and either replace these with that or flesh these out
-  - scoring: rebalance perf metric weightings ([#12577](https://github.com/GoogleChrome/lighthouse/pull/12577))
-  - scoring: update TBT score curve ([#12576](https://github.com/GoogleChrome/lighthouse/pull/12576))
-  - update cumulative-layout-shift ([#12554](https://github.com/GoogleChrome/lighthouse/pull/12554))
-  - update FCP score curve ([#12556](https://github.com/GoogleChrome/lighthouse/pull/12556))
+* The **Performance Category** had a number of scoring changes to align with other performance tools and to better reflect the state of the web. See the [v8.0 Performance FAQ](https://github.com/GoogleChrome/lighthouse/blob/master/docs/v8-perf-faq.md) for more detail.
```

Comment:
I've used the changelog before to go back and pick out prs when notable metric changes have happened. What do you think of still including the PR entries here, something like:

```suggestion
* The **Performance Category** had a number of scoring changes to align with other performance tools and to better reflect the state of the web.
  - The Performance Score has been reweighted ([#12577](https://github.com/GoogleChrome/lighthouse/pull/12577))
  - The TBT and FCP score curves have been updated ([#12576](https://github.com/GoogleChrome/lighthouse/pull/12576), [#12556](https://github.com/GoogleChrome/lighthouse/pull/12556))
  - CLS has been updated to its new, windowed
definition ([#12554](https://github.com/GoogleChrome/lighthouse/pull/12554))

  See the [v8.0 Performance FAQ](https://github.com/GoogleChrome/lighthouse/blob/master/docs/v8-perf-faq.md) for more detail.
```